Source/Scene/ModelExperimental/ModelClippingPlanesPipelineStage.js

```diff
@@ -0,0 +1,125 @@
+import Cartesian2 from "../../Core/Cartesian2.js";
```

Comment:
Do you think we could address https://github.com/CesiumGS/cesium/issues/8234 in this PR since it's just tweaking a sandcastle about clipping planes? I found the Mount St. Helens example to be confusing too.